lib/utils.js

```diff
@@ -0,0 +1,17 @@
+// External dependencies
+var Promise = require( 'bluebird' );
+
+
+/**
+ * Promisify all functions on an object, with a suffix of 'Promise'
+ * e.g. someMethod => someMethodPromise
+ */
+var promisify = function promisify( obj ) {
```

Comment:
Seeing as this method is only used once, is it worth just calling Bluebird directly instead and avoid needed a `utils` file?